compiler/rustc_middle/src/ty/context.rs

```diff
@@ -1281,6 +1281,14 @@ impl<'tcx> TyCtxt<'tcx> {
         if cnum == LOCAL_CRATE { false } else { self.cstore.crate_is_private_dep_untracked(cnum) }
     }
 
+    pub fn original_crate_name(self, crate_num: CrateNum) -> Symbol {
+        if crate_num == LOCAL_CRATE {
+            tcx.crate_name
```

Comment:
This is fine as changing the local crate name will invalidate all incremental caches.

Reply:
Can you make this an in-code comment?